xds/internal/testutils/e2e/clientresources.go

```diff
@@ -97,6 +98,21 @@ func DefaultClientResources(params ResourceParams) UpdateOptions {
 	}
 }
 
+// DefaultClientResourcesWithRLSCSP returns a set of resources (LDS, RDS, CDS, EDS) for a
+// client to connect to a server with a RLS Load Balancer as a child of Cluster Manager.
+func DefaultClientResourcesWithRLSCSP(params ResourceParams, rlsProto *rlspb.RouteLookupConfig) UpdateOptions {
```

Comment:
Does this function need to live here and be exported? Or can it live as an unexported function in the same package in which the test is defined. I don't foresee other tests needing this.